prime-router/src/main/kotlin/Submission.kt

```diff
@@ -25,7 +26,8 @@ import java.util.UUID
 @JsonPropertyOrder(
     value = [
         "id", "submissionId", "timestamp", "sender", "reportItemCount",
-        "errorCount", "warningCount"
+        "errorCount", "warningCount", "status", "plannedCompletionAt",
```

Comment:
Let's put the cool new Luis fields up at the top!  How about this order:  id, submissionid, status, timestamp, plannedCompletionAt, actualComplexiontAt, sender, ....

(I'm thinking it might be nice to group the three dates together)

Reply:
Sounds good!